whats_on/app/routes.js

```diff
@@ -26,6 +27,7 @@ r.get('/exhibitions', renderExhibitions);
 r.get('/exhibitions/coming-up', renderExhibitionsComingUp);
 r.get('/exhibitions/current', renderExhibitionsCurrent);
 r.get('/exhibitions/past', renderExhibitionsPast);
+r.get('/exhibitions/current+coming-up', renderExhibitionsCurrentAndComingUp);
```

Comment:
I think I'd rather either `current-coming-up` or `current-and-coming-up`. Not sure I have any justification for this other than that plus sign looks weird!